### ISSUE TITLE
How do I extract information (consistently) such as Accuracy etc,.. from the distributed example/context ?

### ISSUE THREAD

**Issue by nishantdash97** [tensorflow/tensorflow#29607]
I was using your distribute example: https://www.tensorflow.org/tutorials/distribute/training_loops

I wanted to know how to properly extract some properties such as accuracy in this custom training loop example.

I have tried a couple of things but the documentation regarding this is not too specific.

Something I was initially looking at was:
  - passing in or returning a tf.Keras.metrics object, taking the logits and labels within the step_fn and getting predictions and then accuracy. But I am running into issues of tensor parsing. Should I be using tf.Summary.scalar in some way then?

A concrete reply would help.

**Comment by achandraa** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged//tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there and can provide better help to such issues. Thanks!
